external-dns.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: external-dns
+  version: 0.13.4
+  epoch: 0
+  description: Configure external DNS servers (AWS Route53, Google CloudDNS and others) for Kubernetes Ingresses and Services.
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/kubernetes-sigs/external-dns
+      tag: v${{package.version}}
+      expected-commit: 2905a269102abb653838c27c95d3aa96c05b66e0
+  - runs: |
+      make build
+      mkdir -p ${{targets.destdir}}/usr/bin
+      mv build/external-dns ${{targets.destdir}}/usr/bin/
+update:
```

Comment:
Does `update` know about `git-checkout`? Or will it try to update it to use `fetch`?

Reply:
@rawlingsj said it will tomorrow!

Reply:
yeah let's add the config and we'll just see warnings (soon to be github issues) for failures during the update CRON.  My hope is to sort the `expected-commit` update tomorrow 🤞